src/helpers/discord_utils.ts

```diff
@@ -217,75 +211,115 @@ export async function sendEndRoundMessage(messageContext: MessageContext,
-    const ostPreferenceDisplayStrings = {
-        [OstPreference.INCLUDE]: "Including",
-        [OstPreference.EXCLUDE]: "Excluding",
-        [OstPreference.EXCLUSIVE]: "Exclusively including",
-    };
-    optionStrings[GameOption.OST_PREFERENCE] = `${ostPreferenceDisplayStrings[guildPreference.getOstPreference()]}`;
-    optionStrings[GameOption.RELEASE_TYPE] = `${guildPreference.getReleaseType() === ReleaseType.OFFICIAL ? "only official song releases" : "all songs including dance practices, acoustic versions, remixes, etc"}`;
-    optionStrings[GameOption.MULTIGUESS] = guildPreference.getMultiGuessType() === MultiGuessType.ON ? "Multiple players are able to guess correctly." : "";
+    optionStrings[GameOption.LIMIT] = `${optionStrings[GameOption.LIMIT]} / ${totalSongs.countBeforeLimit}`;
+    optionStrings[GameOption.GENDER] = guildPreference.getGender().join(", ");
+    optionStrings[GameOption.CUTOFF] = `${guildPreference.getBeginningCutoffYear()} - ${guildPreference.getEndCutoffYear()}`;
+    optionStrings[GameOption.ARTIST_TYPE] = guildPreference.getArtistType();
+    optionStrings[GameOption.RELEASE_TYPE] = guildPreference.getReleaseType();
+    optionStrings[GameOption.LANGUAGE_TYPE] = guildPreference.getLanguageType();
+    optionStrings[GameOption.SUBUNIT_PREFERENCE] = guildPreference.getSubunitPreference();
+    optionStrings[GameOption.OST_PREFERENCE] = guildPreference.getOstPreference();
+    optionStrings[GameOption.MULTIGUESS] = guildPreference.getMultiGuessType();
+    optionStrings[GameOption.SHUFFLE_TYPE] = guildPreference.getShuffleType();
+    optionStrings[GameOption.SEEK_TYPE] = guildPreference.getSeekType();
+    optionStrings[GameOption.GUESS_MODE_TYPE] = guildPreference.getGuessModeType();
+    optionStrings[GameOption.TIMER] = guildPreference.isGuessTimeoutSet() ? `${guildPreference.getGuessTimeout()} sec` : null;
+    optionStrings[GameOption.DURATION] = guildPreference.isDurationSet() ? `${guildPreference.getDuration()} mins` : null;
+    optionStrings[GameOption.EXCLUDE] = guildPreference.isExcludesMode() ? guildPreference.getDisplayedExcludesGroupNames() : null;
+    optionStrings[GameOption.INCLUDE] = guildPreference.isIncludesMode() ? guildPreference.getDisplayedIncludesGroupNames() : null;
+
+    const PREFIX = process.env.BOT_PREFIX;
+    const CONFLICT_WARNING_EMOJI = "⚠";
+    const { gameSessions } = state;
+    const isEliminationMode = gameSessions[messageContext.guildID] && gameSessions[messageContext.guildID].gameType === GameType.ELIMINATION;
 
+    // Special case: ,goal is conflicting only when current game is elimination
+    optionStrings[GameOption.GOAL] = guildPreference.isGoalSet() ? String(guildPreference.getGoal()) : NOT_SET_OPTION;
+    optionStrings[GameOption.GOAL] = !isEliminationMode ? optionStrings[GameOption.GOAL] : `${strikethrough(optionStrings[GameOption.GOAL])} (${CONFLICT_WARNING_EMOJI}\`${PREFIX}play ${GameType.ELIMINATION}\` ${CONFLICT})`;
```

Comment:
Write a function to replace ``${strikethrough(optionStrings[GameOption.GOAL])} (${CONFLICT_WARNING_EMOJI}\`${PREFIX}play ${GameType.ELIMINATION}\` ${CONFLICT})`;` so it can be re-used on line 273